libs/data-types/src/lib/utilities/bootstrap-dynamic-types/leverages-union-type.ts

```diff
@@ -0,0 +1,21 @@
+import { ClassSchema, isSimpleCoscradPropertyTypeDefinition } from '../../types';
+
+export const leveragesUniontype = (coscradClassSchema: ClassSchema): boolean => {
```

Comment:
Share this with the team as a classical example of recursion that is implemented with modern es6 syntax.

Reply:
done